src/components/modules/escrow/hooks/initialize-escrow.hook.ts

```
@@ -93,134 +103,85 @@ export const useInitializeEscrow = () => {
   const handleAddMilestone = () => {
     const currentMilestones = form.getValues("milestones");
     const updatedMilestones = [...currentMilestones, { description: "" }];
```

ESLint

[error] 'updatedMilestones' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)